index.bs

```diff
@@ -7136,6 +7141,7 @@ Tobie Langel,
 Kang-Hao Lu,
 Andrea Marchesini,
 Glenn Maynard,
+Isisah Meadows,
 Ms2ger,
```

Comment:
Based on the Github user name, it looks like it should be "Isiah"?

Reply:
Whoops, typo. Will fix, thx.